facturacion_mexico/one_offs/backfill_folio_fiscal.py

```
@@ -0,0 +1,103 @@
+				stats["sin_cambio"] += 1
+				_muestra("sin_cambio", name, current, desired)
+				continue
+
+			categoria = "actualizadas" if desired else "limpiadas"
+			stats[categoria] += 1
+			_muestra(categoria, name, current, desired)
+
+			if apply:
+				# Escritura autoritativa vía el mismo helper del flujo en vivo
+				sincronizar_folio_fiscal(name)
+		except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)